src/main/java/it/gov/pagopa/pu/debtpositions/service/IuvService.java

```diff
@@ -0,0 +1,124 @@
+package it.gov.pagopa.pu.debtpositions.service;
+
+import it.gov.pagopa.pu.debtpositions.exception.InvalidValueException;
+import it.gov.pagopa.pu.organization.dto.generated.Organization;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+/**
+ * Service class exposing methods related to IUV handling.
+ */
+@Lazy
+@Service
+@Slf4j
+public class IuvService {
+
+  public static final String AUX_DIGIT = "3";
+  private static final int CHECK_DIGIT_BASE = 93;
+
+  private final String informationSystemId;
+
+  private final IuvSequenceNumberService iuvSequenceNumberService;
+
+  public IuvService(@Value("${iuv.informationSystemId:00}") String informationSystemId,
```

Comment:
add this property on application.yml
and set the default value directly on the application.yml
```suggestion
  public IuvService(@Value("${iuv.informationSystemId}") String informationSystemId,
```